Client/Frontend/Browser/BrowserViewController/BrowserViewController+TabToolbarDelegate.swift

```diff
@@ -56,6 +56,21 @@ extension BrowserViewController: TabToolbarDelegate, PhotonActionSheetProtocol {
     }
 
     func tabToolbarDidPressMenu(_ tabToolbar: TabToolbarProtocol, button: UIButton) {
+        var whatsNewAction: PhotonActionSheetItem? = nil
+        let showBadgeForWhatsNew = shouldShowWhatsNew()
+        if showBadgeForWhatsNew {
+            // Set the version number of the app, so the What's new will stop showing
+            profile.prefs.setString(AppInfo.appVersion, forKey: LatestAppVersionProfileKey)
```

Comment:
The plan is that when the app menu is shown, that string gets set, and the badge disappears for next time. (Rather than making the user tap the What's New to make the badge disappear).

Reply:
ah okay. I thought that we wanted that behaviour.